tests/serialization/test_design_spaces.py

```diff
@@ -3,13 +3,14 @@
 
 import pytest
 
-from citrine.informatics.design_spaces import DesignSpace, ProductDesignSpace
+from citrine.informatics.descriptors import CategoricalDescriptor, RealDescriptor
+from citrine.informatics.design_spaces import DesignSpace, ProductDesignSpace, EnumeratedDesignSpace
 from citrine.informatics.dimensions import ContinuousDimension, EnumeratedDimension
 
 
 @pytest.fixture
-def valid_data():
-    """Produce valid design space data."""
+def valid_product_data():
```

Comment:
Might be a good place to check out factory-boy. See https://github.com/CitrineInformatics/citrine-python/blob/develop/tests/utils/factories.py

Reply:
That's a good idea. If we don't end up auto-generating these classes we should refactor the orion tests. There's quite a bit of code duplication.